api-server/migrations/20220814155014-change_default_timezone.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.query(`
+      ALTER TABLE datetime_venue
+      ALTER COLUMN timezone
+      SET DEFAULT 'US/Eastern';
+
+      UPDATE datetime_venue
+      SET timezone = 'US/Eastern'
+      WHERE timezone = 'America/New_York';
+    `)
+  },
+
+  down: async (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.query(`
+      ALTER TABLE datetime_venue
+      ALTER COLUMN timezone
+      SET DEFAULT 'America/New_York';
+
+      UPDATE datetime_venue
+      SET timezone = 'America/New_York'
+      WHERE timezone = 'US/Eastern';
```

Comment:
does this really work in reverse? What if someone had entered the timezone as US/Eastern?

Reply:
@chriswininger i don't think i see the problem you see here. in this down-migration, all timezones with a value of `US/Eastern` get rolled back to `America/New_York`, which is correct. this migration just marks a change in the set of timezones and the default that we're using for our instance of the app - to date, everything has been getting the default `America/New_York`, but we decided to change the set we're using. no humans have been able to select a timezone for their datetime so far (and even after they can, they'll only be able to choose from the subset we give them).

or are you asking this from the perspective of "`US/Eastern` and `America/New_York` are not identical"? if that's what you're asking, then the same question goes for the up-migration. this is kind of a can of worms, and we landed on `US/Eastern` for a few reasons, but we can revisit that conversation in this weekend's meeting if you want.